chapter_crashcourse/ndarray.md

```diff
@@ -25,29 +25,29 @@ x = np.arange(12)
 x
 ```
 
-When we print `x`, we can observe the property `<NDArray 12 @cpu(0)>` listed, which indicates that `x` is a one-dimensional array of length 12 and that it resides in CPU main memory. The 0 in `@cpu(0)` has no special meaning and does not represent a specific core.
+When we print `x`, we can observe the property `<ndarray 12 @cpu(0)>` listed, which indicates that `x` is a one-dimensional ndarray of length 12 and that it resides in CPU main memory. The 0 in `@cpu(0)` has no special meaning and does not represent a specific core.
```

Comment:
http://numpy.d2l.ai/chapter_crashcourse/ndarray.html
1. no property `<NDArray 12 @cpu(0)>`
2. explain by default storing in main memory for CPU computing
3. avoid discussing @cpu(0)